faculty.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'school_member'
+
+class Faculty < SchoolMember
+  attr_reader :salary, :target_raise
+
+  def set_phase(num)
+    @phase = num
+    "Cool, I've always wanted to teach phase #{num}!"
+  end
+
+  def salary=(new_salary)
+    puts "This better be good!"
+    @salary = new_salary
+  end
+
+  def receive_raise(raise)
+    @salary += raise
+  end
+
+    def set_performance_rating(rating)
+    response = ""
+    if rating > @rating
+      response = "Yay, I'm a great employee!"
+      receive_raise(@target_raise)
+    else
+      response = "Oh, well -- thanks to this actionable, specific, and kind feedback, I'll do better next time."
+    end
+    response
+  end
```

Comment:
Awesome, great job noticing that these methods were the same except for the target `@rating` and pulling this up.